README.md

```diff
@@ -2147,6 +2149,57 @@ val d: Task[Int] =
 
 > Note: Support for ZIO environments (`R` in `ZIO[R, E, A]`) is currently in development. Once implemented, it will be possible to use ZIO effects with environments directly within Kyo computations.
 
+### Resolvers: GraphQL Server via Caliban
+
+`Resolvers` integrates with the [Caliban](https://github.com/ghostdogpr/caliban) library to help setup GraphQL servers.
+
+The first integration is that you can use Kyo effects inside your Caliban schemas.
+- If your Kyo effects is `(Aborts[Throwable] & ZIOs)` or a subtype of it, a Caliban `Schema` can be derived automatically.
```

Comment:
Might be worth noting that ZIOs includes Fibers/IOs.

Reply:
Added 👍